CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [Unreleased]
+
+## [3.10.0-beta2] 2021-01-29
+
+### Fixed
+
+- Fixed issues with URI when using Routing Policy [PR #1245](https://github.com/3scale/APIcast/pull/1245) [THREESCALE-6410](https://issues.redhat.com/browse/THREESCALE-6410)
```

Comment:
What was the issue with the URI? Please provide details about the original issue. For instance:

 _Fixed issues with the creation of URIs  when using Routing Policy_